### ISSUE TITLE
No Stone Keeper's Shards from H PoS (and possibly other ICC 5 mans)

### ISSUE THREAD

**Issue by arestavo** [dalaranwow/dalaran-wow#5843]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Control Wintergrasp. Kill bosses in ICC5 man HCs such as Pit of Saron. Don't receive Stonekeeper Shards.
**Expected behaviour**: Tell us what should happen instead
Killing any WoTLK 5 man boss, heroic or normal mode, should award Stonekeeper Shards when Wingergrasp is controlled.
**Steps to reproduce the problem**:

1. Own Wintergrasp.
2. Kill bosses in heroic Pit of Saron.
3. Receive no Stonekeeper Shards.

**Include proofs for this behaviour**
See screenshot showing WG ownership/buff, and what the last boss in H PoS awarded.
You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**
![WoWScrnShot_122919_085356](https://user-images.githubusercontent.com/50332138/71559257-35009100-2a19-11ea-8101-f7a72795527f.jpg)

**Comment by arestavo** [comment]
You guys break this stuff a lot after fixing it. Why?

**Comment by arestavo** [comment]
It appears that while in ICC 5 mans, at least heroics, one does not have the Essence of Wintergrasp buff. That doesn't seem right.
